docs/src/tutorials/tut_setup.md

```diff
@@ -1,13 +1,320 @@
+
+# output
+SchoenbergCubicSplineKernel{2}()
+```
+You can find an overview over smoothing kernels and corresponding smoothing
+lengths [here](@ref smoothing_kernel).
+
+For stability, we need numerical dissipation in form of an artificial viscosity
+term.
+```jldoctest tut_setup; output = false
+viscosity = ArtificialViscosityMonaghan(alpha=0.02, beta=0.0)
+
+# output
+ArtificialViscosityMonaghan{Float64}(0.02, 0.0, 0.01)
+```
+We choose the parameters as small as possible to avoid visible viscosity,
+but as large as possible to stabilize the simulation.
+
+The WCSPH method can either compute the particle density by a kernel summation
+over all neighboring particles (see [`SummationDensity`](@ref)) or by making
+the particle density a variable in the ODE system and integrating it over time.
+We choose the latter approach here by using the density calculator
+[`ContinuityDensity`](@ref).
+```jldoctest tut_setup; output = false
+fluid_density_calculator = ContinuityDensity()
+fluid_system = WeaklyCompressibleSPHSystem(tank.fluid, fluid_density_calculator,
+                                           state_equation, smoothing_kernel,
+                                           smoothing_length, viscosity=viscosity,
+                                           acceleration=(0.0, -gravity))
+
+# output
+┌──────────────────────────────────────────────────────────────────────────────────────────────────┐
+│ WeaklyCompressibleSPHSystem{2}                                                                   │
+│ ══════════════════════════════                                                                   │
+│ #particles: ………………………………………………… 360                                                              │
+│ density calculator: …………………………… ContinuityDensity                                                │
+│ correction method: ……………………………… Nothing                                                          │
+│ state equation: ……………………………………… StateEquationCole                                                │
+│ smoothing kernel: ………………………………… SchoenbergCubicSplineKernel                                      │
+│ viscosity: …………………………………………………… ArtificialViscosityMonaghan{Float64}(0.02, 0.0, 0.01)            │
+│ density diffusion: ……………………………… nothing                                                          │
+│ acceleration: …………………………………………… [0.0, -9.81]                                                     │
+│ source terms: …………………………………………… Nothing                                                          │
+└──────────────────────────────────────────────────────────────────────────────────────────────────┘
+```
+
+## Boundary system
+
+In order to define a boundary system, we first have to choose a boundary model,
+which defines how the fluid interacts with boundary particles.
+We will use the [`BoundaryModelDummyParticles`](@ref) with
+[`AdamiPressureExtrapolation`](@ref), which generally produces the best results
+of the implemented methods.
```

Comment:
Since we don't know which methods will still be implemented this should be formulated more neutrally.